config/dynamodb.php

```diff
@@ -39,7 +39,7 @@
         ],
         'aws_iam_role' => [
             'region' => env('DYNAMODB_REGION'),
-            'debug' => true,
+            'debug' => env('DYNAMODB_DEBUG'),
```

Comment:
```suggestion
            'debug' => env('DYNAMODB_DEBUG', env('APP_DEBUG', true)),
```

Maybe is a good idea to have a default value here. By default, the old one.

Reply:
Or it is `true` the default behavior?

Reply:
I think as it was (`env('DYNAMODB_DEBUG')`) is sufficent, otherwise it gets too messy.

Reply:
Yes, seems to work if is set to null.

Reply:
Ok, I fixed also this, but I'm reading that you decided to leave only DYNAMODB_DEBUG.
I fix this again.

Reply:
Fixed!